scripts/judges_server.py

```diff
@@ -0,0 +1,89 @@
+import rospy
+from ros_interop.srv import Team,TeamResponse,Odlc,OdlcResponse
+from ros_interop.msg import *
+
+
+class Teams():
+
+    def __init__(self) -> None:
+        pass
+
+    def router(self,req):
+        type = req.type
+        if type == "GET":
+            return self.get_teams()
+        else:
+            return None
+    def get_teams(self):
+        print("Get Teams Called")
+        teams = []
+        msg = TeamStatus()
+        id = TeamID()
+        id.Id = Id()
+        id.Id.Id = 1
+        id.username = 'hello'
+        id.name = 'pack'
+        id.university = "ncsu"
+        msg.Team = id
+        msg.in_air = False
+        teams = TeamResponse()
+        teams.team_list = []
+        teams.team_list.append(msg)
+        teams.team_list.append(msg)
+        return teams
+
+class odlcs():
```

Comment:
I assume this is going to be for multiple odlc targets, we should make this apart of odlc class and just have two routers in the class.